cell_classification/prepare_data_script_decidua.py

```diff
@@ -0,0 +1,44 @@
+import os
+from segmentation_data_prep import SegmentationTFRecords
+
+os.environ["CUDA_VISIBLE_DEVICES"] = "-1"
+
+
+def naming_convention(fname):
+    return os.path.join(
+        "C:/Users/lorenz/Desktop/angelo_lab/data/decidua/segmentation_data/",
+        fname + "_segmentation_labels.tiff",
+    )
+
+
+data_prep = SegmentationTFRecords(
+    data_dir=os.path.normpath("C:/Users/lorenz/Desktop/angelo_lab/data/decidua/image_data"),
+    cell_table_path=os.path.normpath(
+        "C:/Users/lorenz/Desktop/angelo_lab/data/decidua/"
+        "Supplementary_table_3_single_cells_updated.csv"
+    ),
+    conversion_matrix_path=os.path.normpath(
+        "C:/Users/lorenz/Desktop/angelo_lab/data/decidua/conversion_matrix.csv"
+    ),
+    imaging_platform="MIBI",
+    dataset="decidua_erin",
+    tile_size=[256, 256],
```

Comment:
Is this because you're downsampling by 2x, or want to start out seeing how it works with 256 crops? I think I remember you saying 512 did a bit better, but I could be wrong